documentation/asciidoc/accessories/sense-hat/intro.adoc

```diff
@@ -4,6 +4,6 @@ The https://www.raspberrypi.com/products/sense-hat/[Raspberry Pi Sense HAT] is a
 
 image::images/Sense-HAT.jpg[width="70%"]
 
-The Sense HAT was originally developed for use on the International Space Station as part of the educational https://astro-pi.org/[Astro Pi] programme run by the https://raspberrypi.org[Raspberry Pi Foundation] in partnership with the https://www.esa.int/[European Space Agency]. It is well-suited to many projects that require position, motion, orientation, or environmental sensing. Because it is an add-on board, the Sense HAT connects to a Raspberry Pi device, drawing power from that device.
+The Sense HAT was originally developed for use on the International Space Station as part of the educational https://astro-pi.org/[Astro Pi] programme run by the https://raspberrypi.org[Raspberry Pi Foundation] in partnership with the https://www.esa.int/[European Space Agency]. It is well-suited to many projects that require position, motion, orientation, or environmental sensing. Because it is an add-on board, the Sense HAT draws all necessary power through its GPIO connection to a Raspberry Pi device.
```

Comment:
```suggestion
The Sense HAT was originally developed for use on the International Space Station as part of the educational https://astro-pi.org/[Astro Pi] programme run by the https://raspberrypi.org[Raspberry Pi Foundation] in partnership with the https://www.esa.int/[European Space Agency]. It is well-suited to many projects that require position, motion, orientation, or environmental sensing.
```
...we can probably delete this sentence because it is redundant.
...we can probably delete this sentence because it is redundant.

Reply:
...we can probably delete this sentence because it is redundant.